kubernetes-client/src/test/java/io/fabric8/kubernetes/client/ConfigTest.java

```diff
@@ -201,6 +201,7 @@ private static String decodeUrl(String url) {
   @Test
   public void testWithServiceAccount() {
     System.setProperty(Config.KUBERNETES_KUBECONFIG_FILE, "/dev/null");
+    System.setProperty(Config.KUBERNETES_MASTER_SYSTEM_PROPERTY, "https://kubernetes.default.svc/");
```

Comment:
Here service account could return the different value hence assertion could vary from environment to environment. 

To fix it, Its better way to have deterministic master URL value for SA or let be as this PR to temp fix.

Reply:
Since its blocking the release. Can we let it go and create another issue to fix this?

cc: @iocanel